questionnaires/technology_assessment_measurement_fast_form/technology_assessment_measurement_fast_form_armt.json

```diff
@@ -0,0 +1,866 @@
+        "text_validation_min": "",
+        "text_validation_max": "",
+        "identifier": "",
+        "branching_logic": "",
+        "required_field": "",
+        "custom_alignment": "",
+        "question_number": "",
+        "matrix_group_name": "",
+        "matrix_ranking": "",
+        "field_annotation": "range-type",
+        "evaluated_logic": ""
+    },
+    {
+        "field_name": "tam_usefulness_2",
+        "form_name": "technology_assessment_measurement_fast_form",
+        "section_header": "",
+        "field_type": "range",
+        "field_label": "Overall, I feel the RADAR-CNS system is:",
+        "select_choices_or_calculations": [
+            {
+                "code": "0",
+                "label": "0 Performance enhancing "
+            },
+            {
+                "code": "1",
+                "label": "1 "
+            },
+            {
+                "code": "2",
+                "label": "2 "
+            },
+            {
+                "code": "3",
+                "label": "3 "
+            },
+            {
+                "code": "4",
+                "label": "4 "
+            },
+            {
+                "code": "5",
+                "label": "5 "
+            },
+            {
+                "code": "6",
+                "label": "6 "
+            },
+            {
+                "code": "7",
+                "label": "7 Performance degrading"
+            }
+        ],
+        "field_note": "",
+        "text_validation_type_or_show_slider_number": "",
+        "text_validation_min": "",
+        "text_validation_max": "",
+        "identifier": "",
+        "branching_logic": "",
+        "required_field": "",
+        "custom_alignment": "",
+        "question_number": "",
+        "matrix_group_name": "",
+        "matrix_ranking": "",
+        "field_annotation": "range-type",
+        "evaluated_logic": ""
+    },
+    {
+        "field_name": "tam_usefulness_3",
+        "form_name": "technology_assessment_measurement_fast_form",
+        "section_header": "",
+        "field_type": "range",
+        "field_label": "Overall, I feel the RADAR-CNS system is:",
+        "select_choices_or_calculations": [
+            {
+                "code": "0",
+                "label": "0 Productivity increasing "
+            },
+            {
+                "code": "1",
+                "label": "1 "
+            },
+            {
+                "code": "2",
+                "label": "2 "
+            },
+            {
+                "code": "3",
+                "label": "3 "
+            },
+            {
+                "code": "4",
+                "label": "4 "
+            },
+            {
+                "code": "5",
+                "label": "5 "
+            },
+            {
+                "code": "6",
+                "label": "6 "
+            },
+            {
+                "code": "7",
+                "label": "7 Productivity decreasing"
```

Comment:
A minor detail but would be good to have in
- `field_label`: `Overall, I feel the RADAR-CNS system:`
and the first and last labels as -
- `0  Increases Productivity` and `7 Decreases Productivity`

Reply:
Also the scale is correct? It doesn't need to be likert scale 1-7?